packages/uui-table/lib/uui-table-advanced-example.ts

```diff
@@ -291,3 +291,9 @@ export class UUITableWithSelectionExampleElement extends LitElement {
     `;
   }
 }
+
+declare global {
```

Comment:
I don't think this example should be declared. But I will remove this example code in near future, so I doesn't matter so much :-)